updates/25/updates.sql

```diff
@@ -1,4 +1,808 @@
-UPDATE assets SET identifier="eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943" WHERE identifier="eip155:1/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943";
-INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", "IGU Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", "A", 56, "0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", "IGU", "iguverse-igu", "IGU", NULL, 1667125109, NULL);
-UPDATE evm_tokens SET chain=100 WHERE identifier="eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820";
-INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820", "Safe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820", "A", 100, "0x4d18815D14fe5c3304e87B3FA18318baa5c23820", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820", "SAFE", "safe", "safe", NULL, 1697181885, NULL);
+UPDATE common_asset_details SET swapped_for="eip155:42161/erc20:0x9623063377AD1B27544C965cCd7342f7EA7e88C7" WHERE identifier="eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", "Graph Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", "A", 42161, "0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", "GRT", "the-graph", "grt", NULL, 1623867469, NULL);
+INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", "Impermax", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", "A", 1, "0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", "IMX", "impermax", "", NULL, 1619538925, NULL); 
```

Comment:
why does insert? Are those the missing from the pools?

Reply:
it's for a different asset. I've restored the removed one now